src/main/scala/nl.knaw.dans.easy.dd2d/DdmToDataverseMapper.scala

```diff
@@ -70,6 +71,14 @@ class DdmToDataverseMapper() extends BlockCitation with BlockBasicInformation wi
     addCompoundFieldMultipleValues(temporalSpatialFields, SPATIAL_POINT, ddm \ "dcmiMetadata" \ "spatial" \ "Point", SpatialPoint toEasyTsmSpatialPointValueObject)
     addCompoundFieldMultipleValues(temporalSpatialFields, SPATIAL_BOX, ddm \ "dcmiMetadata" \ "spatial" \ "boundedBy", SpatialBox toEasyTsmSpatialBoxValueObject)
 
+    // Data vault
+    addVaultValue(dataVaultFields, DATAVERSE_PID, deposit.getDataversePid)
```

Comment:
No special function needed. Just juse addPrimitiveFieldSingleValue. The required flag in addVaultValue only forces the service to write the value, even if it is empty, so it doesn't do any validation anyway. And if we do add input validation, it should not be executed in the mapping part.

Reply:
I prefer still to keep this method, because in addPrimitiveFieldSingleValue `NodeSeq` is expected.  But I removed the 'required' parameter.  So, now the value is written only when it is non-empty.